examples/SESAR/test/iSamplesIEEJR000MBasic.json

```diff
@@ -1,5 +1,6 @@
 {
     "$schema": "../iSamplesSchemaBasicSMR.json",
+    "@id": "https://data.isamples.org/digitalsample/igsn/IEEJR000M",
```

Comment:
This was just missing, but I'm pretty sure we wanted to include it as the other sample had it.